zenoh/src/publication.rs

```diff
@@ -902,6 +902,12 @@ impl Priority {
     pub const NUM: usize = 1 + Self::MIN as usize - Self::MAX as usize;
 }
 
+impl From<zenoh_protocol::core::Priority> for Priority {
```

Comment:
This is not safe. `zenoh_protocol::core::Priority::Control` does not exist in `Priority`, so the mapping will not be possible in that case. It should be a `TryFrom`.

Reply:
fixed